webapp-django/crashstats/manage/templates/admin/debug_view.html

```diff
@@ -0,0 +1,35 @@
+{% extends 'admin/base_site.html' %}
+{% load i18n admin_urls static admin_list %}
+
+{% if not is_popup %}
+    {% block breadcrumbs %}
+        <div class="breadcrumbs">
+            <a href="{% url 'admin:index' %}">{% trans 'Home' %}</a>
+            &rsaquo; Management pages
+            &rsaquo; Debug information
+        </div>
+    {% endblock %}
+{% endif %}
+
+{% block content %}
+    <div id="content-main">
+        <div>
+            <table>
+                <thead>
+                    <tr>
+                        <th>Key</th>
+                        <th>Value</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    {% for key, val in debug_info.items %}
+                        <tr class="row1">
```

Comment:
What does `class="row1"` do?

Reply:
This is a Django admin template, so I've been using the classes Django defined. When I did the Site Status page, I figured out which classes to use to get close to what I needed. But that was a while ago and I don't remember why I did what I did. I should probably write up a style guide. Either that or we should switch to our own style classes--probably a better idea, but probably not for this PR.